google/resource_bigtable_gc_policy.go

```diff
@@ -62,9 +63,15 @@ func resourceBigtableGCPolicy() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"days": {
 							Type:        schema.TypeInt,
-							Required:    true,
+							Optional:    true,
+							Deprecated:  "Deprecated in favor of seconds",
 							Description: `Number of days before applying GC policy.`,
 						},
+						"seconds": {
```

Comment:
We will want to add `ExactlyOneOf` to the schema to ensure one of these are set. `ExactlyOneOf` must use hard coded resource indexes to work with lists, so we will want to limit this max_age list to a length of 1 (this is effectively how it works anyways)

Example: https://github.com/hashicorp/terraform-provider-google/blob/master/google/resource_compute_resource_policy.go#L142

Reply:
After this is added you can remove the log around "Either seconds or days must be set". Not setting either will cause a panic before that error is hit anyways